src/app/downloads/downloads-table.component.ts

```diff
@@ -131,11 +81,25 @@ export class DownloadsTableComponent implements OnDestroy {
     this.podcastMetricsStoreSub = this.store.select(selectPodcastMetrics).subscribe((podcastMetrics: PodcastMetricsModel[]) => {
       this.podcastMetrics = findPodcastMetrics(this.filter, podcastMetrics);
       if (this.podcastMetrics) {
+        if (!this.podcastMetrics.allTimeDownloads) {
+          this.store.dispatch(new CastlePodcastAllTimeMetricsLoadAction({ filter: this.filter }));
```

Comment:
Soliciting advice on this front: sometimes when the `CastlePodcastAllTimeMetricsLoadAction` is fired (from the ngOnInit and the constructor), `this.podcasts` is not yet defined in the Castle Effects. So I'm reissuing the fetch here if we get to this point but still don't have all time downloads.